test-data/unit/check-classes.test

```diff
@@ -3725,6 +3725,7 @@ class C(B):
     a = "a"
 [out]
 main:4: error: Incompatible types in assignment (expression has type "str", base class "A" defined the type as "int")
+main:6: error: Incompatible types in assignment (expression has type "str", base class "A" defined the type as "int")
```

Comment:
This looks like a false positive. It can cause errors in code that already passes mypy.

I assume this is related to the `break` statement you removed.

Reply:
Yes, it's related to the `break` statement being removed.

I wouldn't say that it is a false positive. `C.a` is incompatible with `A.a`, just as `B.a` is incompatible with `A.a`.

Reply:
Mypy simply would not show you the error before. If you fixed the compatibility problem in `B.a`, then you would get an error about `C.a`. Isn't it unexpected to start getting new errors when mypy could have reported it the first time?